config/internal/apiserver/sample-pipeline/sample-pipeline.yaml.tmpl

```diff
@@ -72,16 +63,16 @@ data:
             container:
               args:
               - --executor_input
-              - '{{"{{"}}${{"}}"}}'
+              - '{{$}}'
               - --function_to_execute
               - create_dataset
               command:
               - sh
               - -c
               - "\nif ! [ -x \"$(command -v pip)\" ]; then\n    python3 -m ensurepip ||\
                 \ python3 -m ensurepip --user || apt-get install python3-pip\nfi\n\nPIP_DISABLE_PIP_VERSION_CHECK=1\
-                \ python3 -m pip install --quiet     --no-warn-script-location 'kfp==2.0.1'\
-                \ && \"$0\" \"$@\"\n"
+                \ python3 -m pip install --quiet     --no-warn-script-location 'pandas==2.2.0'\
+                \ 'kfp==2.0.0-beta.13' && \"$0\" \"$@\"\n"
```

Comment:
is this compiled with an ancient kfp sdk? 2.7.0 should work

Reply:
was trying to see why metrics output artifact had magically disappeared.  turns out the pipeline in `config/internal/apiserver/sample-pipeline.yaml.tmpl` (which was compiled with the now-defunct kfp-2.0.1) was mismatched with the one in `docs/`.  This and the mismatch have now been updated to maintain sync